va/va.go

```diff
@@ -124,6 +125,7 @@ func NewValidationAuthorityImpl(
 	stats metrics.Scope,
 	clk clock.Clock,
 	logger blog.Logger,
+	accountURIPrefixes []string,
```

Comment:
Can we add a check here for `features.Enabled(features.CAAAccountURI) && len(accountURIPrefixes) == 0` and error out, otherwise we'd fail all validations.

Reply:
Done.